src/docs/asciidoc/installing_upgrading.adoc

```diff
@@ -543,7 +543,7 @@ It is worth mentioning that a business app upgrade is orthogonal to a rolling me
 === Running in Modular Java
 
 Java link:http://openjdk.java.net/projects/jigsaw/[project Jigsaw^] brought
-a new Module System into Java 9 and newer. Hazelcast supports running in
+a new Module System into Java 9 and later. Hazelcast supports running in
```

Comment:
Probably want to update to Java 11 to be consistent.

Reply:
Updating this to Java 11 would make it incorrect. Maybe we can say `If you're using JDK 11 or later, you can use Hazelcast as a module in the [Java Platform Module System](link to more info).`